src/buildblock/ProjDataFromStream.cxx

```diff
@@ -194,6 +194,96 @@ ProjDataFromStream::get_viewgram(const int view_num, const int segment_num,
   return viewgram;    
 }
 
+float
+ProjDataFromStream::get_bin_value(const Bin& this_bin) const
+{
+    return get_bin_value(this_bin.segment_num(),
+                         this_bin.axial_pos_num(),
+                         this_bin.view_num(),
+                         this_bin.tangential_pos_num());
+}
+
+
+float
+ProjDataFromStream::get_bin_value(const int segment_num,
```

Comment:
do we need the version with 4 ints? Scope for confusion in argument order. (Of course it's there in Bin as well, but at least it's in 1 place)